packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/PayableDetailsForm.tsx

```diff
@@ -303,7 +347,10 @@ const PayableDetailsFormBase = forwardRef<
                             fullWidth
                             error={Boolean(error)}
                             helperText={error?.message}
-                            required
+                            required={isFieldRequired(
```

Comment:
You have to mark fields as required only if `payable.ocr_status` is empty (it's `null`, but OpenAPI doesn't support nulls). For normal payables this check is not needed.